electrum/tests/test_transaction.py

```diff
@@ -58,8 +58,12 @@ def test_bytes(self):
         s.write(b'foobar')
         self.assertEqual(s.read_bytes(3), b'foo')
         self.assertEqual(s.read_bytes(2), b'ba')
-        self.assertEqual(s.read_bytes(4), b'r')
-        self.assertEqual(s.read_bytes(1), b'')
+        with self.assertRaises(transaction.SerializationError):
+            self.assertEqual(s.read_bytes(4), b'r')
```

Comment:
The `assertEqual` does not really make sense here.
```suggestion
            s.read_bytes(4)
```

Reply:
fixed in 3cc5756